IGCexpansion/CodonGeneconv.py

```diff
@@ -220,6 +220,16 @@ def get_initial_x_process(self, transformation = 'log'):
             self.x_rates = np.exp(-np.exp(self.x_rates))
         self.x = np.concatenate((self.x_process, self.x_rates))
 
+        if self.Force is not None:
+            self.Force.update({i:np.exp(self.logzero) for i in self.Force if self.Force[i] == 0})
+            for i in self.Force.keys():
+                if transformation == 'log':
+                    self.x[i] = np.log(self.Force[i])
+                elif transformation == 'None':
+                    self.x[i] = self.Force[i]
+                elif transformation == 'Exp_Neg':
+                    self.x[i] = -np.exp(self.Force[i])

```

Comment:
If the purpose of this pull request is to override the `summary` output, then please just change the related function for generating the `summary` but not here.  These dimensions in `self.x` are masked by `self.Force` anyway and your code here generate `log(0)` issue which is why `self.x` value was not changed when I wrote the code.

Reply:
The purpose of this pull request is to change the printed `Current x array` in the python console. I understand it now. We can just change the part when the program prints the `Current x array`.

Reply:
In this case, I suggest adding a line for `Force` information in the `print to screen`.